RevenueCat/Plugins/Editor/RevenueCatDependencies.xml

```diff
@@ -1,10 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <dependencies>
     <androidPackages>
-        <androidPackage spec="org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.2.61" />
```

Comment:
is this not needed anymore?

Reply:
yeah, this comes transitively from the common package. I am not sure why we had this 🤔